src/CppUTest/TestOutput.cpp

```diff
@@ -145,14 +145,20 @@ void TestOutput::printCurrentGroupEnded(const TestResult& /*res*/)
 void TestOutput::printTestsEnded(const TestResult& result)
 {
     print("\n");
-    const bool anyTestFailed = result.getFailureCount() > 0;
-    if (anyTestFailed) {
+    const bool isFailure = result.isFailure();
+    if (isFailure) {
         if (color_) {
             print("\033[31;1m");
         }
         print("Errors (");
-        print(result.getFailureCount());
-        print(" failures, ");
+        const int failureCount = result.getFailureCount();
+        if (failureCount > 0) {
+            print(failureCount);
+            print(" failures, ");
+        }
+        else {
+            print("ran nothing, ");
```

Comment:
Can we make the error much more descriptive for the failure because no tests run? I think this might be a surprise to people and I think the error ought to explain that. Perhaps:

"Test run failed because no tests were run or ignored. Assuming something went wrong"

Reply:
I think, better to put this as a note after the statistic to keep it readable. Maybe even new line.
Or do you want it to be printed right next to `Errors` before the stats?

Reply:
A new line will be fine. At least, the reason for failing needs to be very clear as it will come as a surprise to people.

Reply:
Done